Dangerfile

```diff
@@ -0,0 +1,11 @@
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+warn("PR is classified as Work in Progress. When complete, please remove the [WIP] and assign reviewers.") if github.pr_title.include? "[WIP]"
+
+warn "Please link to a GitHub issue within this pull request. Example: Closes #9999" unless (github.pr_body.include? "Closes #" or github.pr_body.include? "References #")
+
+# Add in our standard using GitHub Closes within the Pull request body.
+warn "Please add a Changes section to your pull request description." unless github.pr_body.include? "Changes"
+warn "Please add a Purpose section to your pull request description." unless github.pr_body.include? "Purpose"
+
+# Warning if the user has not assigned themselves to the Pull Request.
+warn "This PR does not have any assignees yet. You may want to assign yourself if this is your Pull Request." unless github.pr_json["assignee"]
```

Comment:
I would be consistent about "Pull Request" vs "pull request".

Reply:
Thanks, good idea. 

Fixed via: 0eb3fe0b3868536112e78c5f68e0d6617a5e719f